rmgpy/molecule/draw.py

```diff
@@ -557,6 +560,8 @@ def _generate_ring_system_coordinates(self, atoms):
                     if count == 1 or count == 2:
                         if cycle is None or len(cycle0) > len(cycle): cycle = cycle0
             cycle0 = cycle1
+            if cycle is None:
+                break
```

Comment:
could you explain this addition?

Reply:
Without it trying to draw
```
1  O u0 p2 c0 {5,S} {8,S}
2  O u0 p2 c0 {6,S} {7,S}
3  O u0 p3 c-1 {6,S} {8,H}
4  O u0 p2 c0 {6,D} {9,H}
5  N u0 p1 c0 {1,S} {7,H} {9,S} {10,S}
6  N u0 p0 c+1 {2,S} {3,S} {4,D}
7  H u0 p0 c0 {2,S} {5,H}
8  H u0 p0 c0 {1,S} {3,H}
9  H u0 p0 c0 {4,H} {5,S}
10 H u0 p0 c0 {5,S}
```
leads to
```
Traceback (most recent call last):
  File "/Users/rwest/Code/RMG-Py/rmgpy/molecule/draw.py", line 209, in draw
    self._generate_coordinates()
  File "/Users/rwest/Code/RMG-Py/rmgpy/molecule/draw.py", line 366, in _generate_coordinates
    self._generate_ring_system_coordinates(backbone)
  File "/Users/rwest/Code/RMG-Py/rmgpy/molecule/draw.py", line 564, in _generate_ring_system_coordinates
    atoms.remove(cycle)
ValueError: list.remove(x): x not in list
```

If you replace the `break` with a `continue` then you enter and endless loop (due to the `while`)

With the `break` we get the image
![image](https://user-images.githubusercontent.com/93807/84344219-36151d00-ab78-11ea-816d-8bfbf14cc0e5.png)

...which is not great - the H is being drawn exactly on top of an O which hides it - but at least it doesn't crash.